inspirehep/modules/workflows/tasks/arxiv.py

```diff
@@ -233,9 +233,9 @@ def arxiv_author_list(stylesheet="authorlist2marcxml.xsl"):
     def _author_list(obj, eng):
         arxiv_id = get_arxiv_id(obj.data)
         filename = secure_filename('{0}.tar.gz'.format(arxiv_id))
-        tarball = obj.files[filename]
-
-        if not tarball:
+        try:
+            tarball = obj.files[filename]
+        except KeyError:
```

Comment:
Just curious, was this an api change? Can `tarball` still be `None`? Should we check for that too?

Reply:
AFAIK ``obj.files`` item it doesn't make sense to be ``None``, it should be there or not. I didn't dig in more on this but I can check how we get the files into the workflow object

Reply:
so you cannot have ``None`` it reads directly from the bucket

Reply:
Not the `obj.files` item itself, but whatever it's `__getitem__` returns.

Reply:
Hmm it'll give you a ``KeyError`` actually https://github.com/inveniosoftware/invenio-records-files/blob/master/invenio_records_files/api.py#L159-L164

Reply:
and I just saw, this is how we handle the plots actually https://github.com/inspirehep/inspire-next/blob/b40a21e9bfc92922ed88edef801cd7ef07187e05/inspirehep/modules/workflows/tasks/arxiv.py#L126-L139

Reply:
:+1:

Reply:
if it's ok, could you please also approve it 🤓